conf/messages.hr

```diff
@@ -118,6 +118,8 @@ no_inspection_needed_page_export.no_controls.p1=Možete nastaviti prema graničn
 # Inspection pending
 #
 ###############################################
+inspection_pending_page.notification.title=Važno
+inspection_pending_page.notification.p1=Nije došlo do promjene statusa inspekcije
 inspection_pending_page.title=Vaš status inspekcije još nije spreman
 inspection_pending_page.p1=Vaš status inspekcije trebao bi biti spreman približno 10 minuta prije nego što stignete na granični prijelaz dolaska. S pomoću gumba u nastavku možete ponovno provjeriti je li status spreman.
```

Comment:
Do the same keys need deleting in this file too for Croatian?

Reply:
For a second I thought it was @Mahesh49 commenting 😆 Well spotted. Thank you @tipperba

Reply:
I've updated the PR